frontend/src/components/shared/button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './button.scss';
+import { Link } from 'react-router-dom';
+
+type ButtonProps = {
+  name: string;
+  style: string;
+};
+
+const Button: React.FC<ButtonProps> = props => {
+  return (
+    <Link to="/login" className={props.style}>
```

Comment:
shared and abstract button which will always point to login? maybe it is better to rename it or make more abstract, also it is not a button, but Link which Maybe looks like a button and depend on props.